src/puppeteer_crawler.js

```diff
@@ -258,6 +217,7 @@ class PuppeteerCrawler {
             maxRequestRetries,
             maxRequestsPerCrawl,
             handleRequestFunction: (...args) => this._handleRequestFunction(...args),
+            handleRequestTimeoutSecs: handlePageTimeoutSecs * 2,
```

Comment:
Or we could disable this timeout when BasicCrawler is used by PuppeteerCrawler because Puppeteer crawler has own timeout for user input function (handlePageFunctionTimeout). And by having this another timeout wrapping our own code we can miss some bugs causing some of our functions to timeout. Ie. 

* when user uses BasicCrawler then this timeout definitely makes sense and is needed
* but when user uses PuppeteerCrawler then his code is already wrapped by one timeout and this one is not needed so I would set it to 0 (=no timeout)

Reply:
I think that we should keep the timeout even when PuppeteerCrawler is used, because in a typical scenario, it should never fire. If it fires, we will find it in the logs, so we will know that something is going on. It will just keep actors from hanging forever.